docs/rules/attribute-names.md

```diff
@@ -33,6 +36,45 @@ The following patterns are not warnings:
 @property({attribute: 'camel-case-name'})
 camelCaseName: string;
 
+@property({attribute: 'camel-case-other-name'})
+camelCaseName: string;
+
+@property()
+lower: string;
+```
+
+## Options
+
+You can specify `style` to one of these values `none`, `snake`, `kebab` to
+enforce that attribute name is the styled version of property, or `false`.
```

Comment:
it could be read as you being able to set `style: false` i think.

```suggestion
enforce that the attribute name is cased using the specified casing style.
```